.gitignore

```diff
@@ -116,3 +116,6 @@ dist
 
 #removing package-lock as the master branch has yarn.lock
 package-lock.json
+
+# Jetbrains ide config files
+.idea
```

Comment:
This is the first time I've seen someone use Jetbrains as an IDE for JS/TS.

Reply:
I find WebStorm very convenient :D